portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,16 +35,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static class Comment {
-    private String email, comment;
-    public Comment(String email, String comment){
+    private String email, comment, time;
+    public Comment(String email, String comment, String time){
       this.email = email;
       this.comment = comment;
+      this.time = time;
     }
   }
-  private List<Comment> comments = new ArrayList<Comment>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment").addSort("time", SortDirection.DESCENDING);;
```

Comment:
Pick a descriptive name ("query" isn't very descriptive, it's the name of the type).